### PR TITLE
Sofb fixes

### PR DESCRIPTION
**Pull request by fernandohds564**
- add peak-to-peak info in orbit stats;
- add option to load orbit differences in registers;
- add `ref_orb` shortcut in RefOrb setting widget.

### CODE REVIEW

File: pyqt-apps/siriushla/as_ap_sofb/orbit_register.py [hunk 1]
```diff
@@ -105,7 +105,7 @@ def get_registers_control(self):
 class OrbitRegister(QWidget):
     """Create the Context Menu for the Registers."""
 
-    DEFAULT_DIR = '/home/fac/sirius-iocs/si-ap-sofb'
+    DEFAULT_DIR = '/home/sirius/screens-iocs'
```
Comment (xresende) [inline pyqt-apps/siriushla/as_ap_sofb/orbit_register.py:108]: The official (managed by ansible) path is `/home/sirius/mounts/screens-iocs`

File: pyqt-apps/siriushla/as_ap_sofb/graphics/base.py [hunk 2]
```diff
@@ -19,14 +19,15 @@
 
 class BaseWidget(QWidget):
     """."""
-    DEFAULT_DIR = _pathlib.Path.home().as_posix()
-    EXT = '.txt'
-    EXT_FLT = 'Text Files (*.txt)'
+
+    DEFAULT_DIR = '/home/sirius/screens-iocs'
```
Comment (xresende) [inline pyqt-apps/siriushla/as_ap_sofb/graphics/base.py:23]: The official (managed by ansible) path is `/home/sirius/mounts/screens-iocs`

File: pyqt-apps/siriushla/as_ap_sofb/orbit_register.py (hunk 1 above)
Comment (fernandohds564) [reply pyqt-apps/siriushla/as_ap_sofb/orbit_register.py]: I fixed it @xresende, thanks!

File: pyqt-apps/siriushla/as_ap_sofb/graphics/base.py (hunk 2 above)
Comment (fernandohds564) [reply pyqt-apps/siriushla/as_ap_sofb/graphics/base.py]: done!

**Pull request merged**